### ISSUE TITLE
graph isomorphism

### ISSUE THREAD

**Issue by pfps** [w3c/N3#199]
As far as I can tell, graph isomorphism is claimed to be symmetric without proof.

As far as I can tell, graph isomorphism is not claimed to be reflexive or transitive.

I suspect that there should be proofs that graph isomorphism is indeed symmetric, reflexive, and transitive.

**Comment by doerthe** [comment]
You are right. We plan to add these and a few other proofs for the next release.  

In the mean time we encourage you to ask for all the details (proofs) you would be interested in.

Thank you for your comments.
